### ISSUE TITLE
Consolidate puphpet-release-composer-installer here?

### ISSUE THREAD

**Issue by beporter** [loadsys/composer-plugins#3]
There isn't really a big reason to have multiple "composer plugin" style repos when they can all live together. The performance hit in including this single project (that includes unneeded or unrelated composer tooling) in a bunch of other projects is minimal, but there's some savings in being able to "just include our composer plugins" and get a bunch of tooling all at once. (It gets to be a lot of administrative overhead to manage so many small packages.)

So basically: It'd be safe to relocate all of the code from the [puphpet-release-composer-installer](https://github.com/loadsys/puphpet-release-composer-installer) project here and update the `puphpet-release` composer.json to just require `loadsys/composer-plugins` instead.
